driver-core/src/main/java/com/datastax/driver/core/TypeCodec.java

```diff
@@ -1079,6 +1093,11 @@ private static Long parseTime(String s) throws IllegalArgumentException {
 
         @Override
         public Long parse(String value) {
+            // enclosing single quotes required, even for long literals
```

Comment:
Noticed that the other codecs simply strip quotes while this one requires quotes and then strips them.  Is this because of what you documented in [JAVA-817](https://datastax-oss.atlassian.net/browse/JAVA-817) about values exceeded Integer.MAX_VALUE?